homekit/controller/ble_impl/__init__.py

```diff
@@ -592,10 +592,15 @@ def find_characteristic_by_uuid(device, service_uuid, char_uuid):
         if ServicesTypes.get_short(service_uuid.upper()) == ServicesTypes.get_short(possible_service.uuid.upper()):
             service_found = possible_service
             break
-    logger.debug('searched service: %s', service_found)
+    logger.debug('searched service: %s', service_found.uuid)
 
     if not service_found:
-        logging.error('searched service not found.')
+        try:
+            srv_desc = f'{Service[service_uuid]} '
```

Comment:
Service is coming from https://github.com/kvaellning/homekit_python/blob/ble_pairing_strategy/homekit/controller/ble_impl/gatt.py#L80 ?

Reply:
Yeah, the service types are meant